buildenvs/xen.Dockerfile

```
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2024, Unikraft GmbH and The KraftKit Authors.
+# Licensed under the BSD-3-Clause License (the "License").
+# You may not use this file except in compliance with the License.
+
+FROM kraftkit.sh/base-golang:latest
+
+RUN set -xe; \
+    apt-get update; \
+    apt-get install -y \
+        binutils \
+        bison \
+        build-essential \
+        cmake \
+        flex \
+        gcc \
+        git \
+        iasl \
+        libglib2.0-dev \
+        libncurses5-dev \
+        libpixman-1-dev \
+        libslirp-dev \
+        libssh2-1-dev \
+        libssl-dev \
+        libyajl-dev \
+        make \
+        ninja-build \
+        perl \
+        pkg-config \
+        python3 \
+        python3-pip \
+        python3-setuptools \
+        python3-wheel \
+        uuid-dev; \
+    pip3 install python-config --break-system-packages; \
+    git clone -b stable-4.18 https://xenbits.xen.org/git-http/xen.git /xen; \
```

can you turn this version into an `ARG`?

```suggestion
    git clone -b stable-${XEN_VERSION} https://xenbits.xen.org/git-http/xen.git /xen; \
```